week-2/L-array-methods-2/exercise3.js

```diff
@@ -7,15 +7,20 @@
 var ukNations = ["Scotland", "Wales", "England", "Northern Ireland"];
 
 function isInUK(country) {
-  return; // complete this statement
+  for(var i = 0; i < ukNations.length; i++){
+    if(country == ukNations[i]){
```

Comment:
There's a function already that you can use on arrays to find an element. Probably better to use that instead of writing your own code!